### PR TITLE
update testing files for mo_photo

### PR DESCRIPTION
**Pull request by tangshuaiqi**
1. regenerate testing files for timestep 353 to cover some code and write out all levels of output for cloud_mod.
2. save testing data for all four columns for table_photo

### CODE REVIEW

File: components/eam/src/chemistry/yaml/mo_photo/f90_yaml/table_photo_beg_yml.f90 [hunk 1]
```diff
@@ -86,14 +86,14 @@
...
         
-        call write_var(unit_input,unit_output,'pmid',pmid(yaml%col_print,yaml%lev_print))
-        call write_var(unit_input,unit_output,'pdel',pdel(yaml%col_print,yaml%lev_print))
-        call write_var(unit_input,unit_output,'temper',temper(yaml%col_print,yaml%lev_print))
-        call write_var(unit_input,unit_output,'col_dens',col_dens(yaml%col_print,yaml%lev_print,:))
-        call write_var(unit_input,unit_output,'zen_angle',zen_angle(yaml%col_print))
-        call write_var(unit_input,unit_output,'srf_alb',srf_alb(yaml%col_print))
-        call write_var(unit_input,unit_output,'lwc',lwc(yaml%col_print,yaml%lev_print))
-        call write_var(unit_input,unit_output,'clouds',clouds(yaml%col_print,yaml%lev_print))
+        call write_var(unit_input,unit_output,'pmid',pmid(:,yaml%lev_print))
```
Comment (odiazib) [inline components/eam/src/chemistry/yaml/mo_photo/f90_yaml/table_photo_beg_yml.f90]: can we save all `kk` values? 

i.e.,`call write_var(unit_input,unit_output,'pmid',pmid(:,:))`

Comment (odiazib) [reply components/eam/src/chemistry/yaml/mo_photo/f90_yaml/table_photo_beg_yml.f90]: same comment for the rest of  variables.

File: components/eam/src/chemistry/yaml/mo_photo/table_photo_input_ts_355.yaml [hunk 2]
```diff
@@ -4,13 +4,13 @@ mam4xx:
...
     dt:    0.00
-    pmid: [0.6949830380E+005]
-    pdel: [0.3530697329E+004]
-    temper: [0.2764186747E+003]
-    col_dens: [0.6303476666E+019,0.3094836910E+025]
-    zen_angle: [0.1350784017E+001]
-    srf_alb: [0.6450720099E+000]
-    lwc: [0.1035073817E-030]
-    clouds: [0.0000000000E+000]
+    pmid: [0.6949830380E+005,0.6753476429E+005,0.6902605804E+005,0.6775109514E+005]
```
Comment (odiazib) [inline components/eam/src/chemistry/yaml/mo_photo/table_photo_input_ts_355.yaml]: please save these two variables : `pht_alias_mult`, `lng_indexer`

**Review by odiazib — CHANGES_REQUESTED** [review]

**Review by odiazib — APPROVED** [review]

**Comment by odiazib** [comment]
It looks great! Thanks

**Comment by tangshuaiqi** [comment]
All tests passed. merge into maint

comp_testall_photo_fix_mam4_org_v2_baselines: 9 tests
  PASS ERP_P64x1.ne4pg2_oQU480.F2010.compy_intel
  PASS ERS.ne4pg2_oQU480.F2010.compy_intel
  PASS PEM_P64x1.ne4pg2_oQU480.F2010.compy_intel
  PASS PET.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_D_Ln5_P1x1.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_D_Ln5_P32x1.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_D.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_Ly1.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS.ne30pg2_EC30to60E2r2.F2010.compy_intel

**Pull request merged**